packages/utilities/src/rtl.ts

```diff
@@ -1,46 +1,55 @@
 import { KeyCodes } from './KeyCodes';
+import { getDocument } from './dom';
+import { getItem, setItem } from './localStorage';
+
+const RTL_LOCAL_STORAGE_KEY = 'isRTL';
 
 // Default to undefined so that we initialize on first read.
 let _isRTL: boolean | undefined;
 
 /**
  * Gets the rtl state of the page (returns true if in rtl.)
- *
- * @public
  */
 export function getRTL(): boolean {
-  let isRTL: boolean = _isRTL as boolean;
+  if (_isRTL === undefined) {
+    let doc = getDocument();
+
+    let savedRTL = getItem(RTL_LOCAL_STORAGE_KEY);
```

Comment:
i'm nervous about this. if local storage setting is set, then we always use that in every product, overriding the dir on the html element.

And, coupled with actually writing it here, if a product changes languages via account settings and burns on a new dir, surprise, it won't flip, because we wrote it on initial access of getRTL. That would be a difference in behavior than we have today and it would make a lot of people upset.

I would suggest instead; 

1. don't write this on the read.
2. only write this value from the demo page.

Reply:
Done. I can't do the second suggestion until this PR is merged because of circular dependencies